src/parsers/perf/perfparser.cpp

```diff
@@ -811,8 +811,27 @@ struct PerfParserPrivate
         auto args = features.cmdline;
         args.removeFirst();
         summaryResult.command = QLatin1String("perf ") + QString::fromUtf8(args.join(' '));
+        summaryResult.hostName = QString::fromUtf8(features.hostName);
+        summaryResult.linuxKernelVersion = QString::fromUtf8(features.osRelease);
+        summaryResult.perfVersion = QString::fromUtf8(features.version);
+        summaryResult.cpuDescription = QString::fromUtf8(features.cpuDesc);
+        summaryResult.cpuId = QString::fromUtf8(features.cpuId);
+        summaryResult.cpuArchitecture = QString::fromUtf8(features.arch);
+        summaryResult.cpusOnline = features.nrCpusOnline;
+        summaryResult.cpusAvailable = features.nrCpusAvailable;
+        summaryResult.cpuSiblingCores = QString::fromUtf8(features.siblingCores.join(", "));
+
+        QList<QByteArray> siblingThreads;
```

Comment:
Looking at the output from perf again, I think I misunderstood this and only realize it now that I see how you parse this here. So, sorry, but this needs to be revised again:

the sibling threads seems to indicate hardware threads, i.e. CPU ids, that share the same core. So for my Intel 8 core e.g. I have:

- siblingCores=(0-7)
- siblingThreads=(0,4, 1,5, 2,6, 3,7)

this means, I have one CPU node with 8 cores on them, but actually two of them always share the same hardware (hyperthreading). So I have only 4 "real" cores, and 8 "virtual" ones.

So splitting and sorting like I proposed is very wrong. Sorry again. Instead, I'd suggest we change the format to something like this:

Sibling Cores: [0-7]
Sibling Threads: [0,4], [1,5], [2,6], [3,7]

So for both of these lists, wrap every entry in '[' and ']', then join everything together with commas.